src/pwr.ps1

```diff
@@ -1192,10 +1325,21 @@ switch ($Command) {
 			}
 		}
 	}
+	'prune' {
+		if ($Packages.Count -gt 0) {
+			Write-PwrFatal 'prune not compatible with -Packages'
+		}
+		Lock-PwrLock
+		try {
+			Prune-PwrPackages
+		} finally {
+			Unlock-PwrLock
+		}
+	}
 	{$_ -in 'rm', 'remove'} {
 		Lock-PwrLock
 		try {
-			if ($DaysOld) {
+			if ($DaysOld) { # TODO: Deprecate and remove eventually
```

Comment:
Let me know what you think we should do here. Here's some options I thought of:
- Do nothing and leave the comment.
- Add a deprecation message anytime this option is run.
- Remove the `-DaysOld` option processing logic (and associated documentation).
- Remove the comment and keep it as-is.

Reply:
I like adding a deprecated warning and removing it on a minor/major update.

Reply:
Done.